.github/workflows/ci-test-suite.yml

```
@@ -191,12 +196,163 @@ jobs:
+        uses: ./.github/actions/docker-tag-push
+        with:
+          local_image: frontend-amd64:${{ github.run_id }}
+          push_tags: |
+            ${{ env.REGISTRY_IMAGE }}:${{ env.IMAGE_PREFIX }}-frontend-amd64
+            ${{ env.REGISTRY_IMAGE }}:${{ env.IMAGE_PREFIX }}-frontend-amd64-run-${{ github.run_id }}
+          aws_push: 'true'
+          azure_push: 'true'
+          aws_account_id: ${{ secrets.AWS_ACCOUNT_ID }}
+          aws_default_region: ${{ secrets.AWS_DEFAULT_REGION }}
+          azure_acr_hostname: ${{ secrets.AZURE_ACR_HOSTNAME }}
+          azure_acr_user: ${{ secrets.AZURE_ACR_USER }}
```

actionlint (1.7.10)

input "azure_acr_user" is not defined in action "Docker Tag and Push" defined at "./.github/actions/docker-tag-push". available inputs are "aws_account_id", "aws_default_region", "aws_push", "azure_acr_hostname", "azure_push", "conditional_tag", "local_image", "push_tags"

(action)